test/lib/components/TimeSeriesSlider/TimeSeriesSlider.test.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import TimeSeriesSlider from '../../../../src/lib/components/TimeSeriesSlider/TimeSeriesSlider.js'
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import configureStore from 'redux-mock-store';
+import periodData from '../../../fixtures/periodData.json';
+import { layer1 } from '../../../fixtures/defaultLayers';
+import sampleLayer from '../../../fixtures/sample-layer.json'
+var gisida = require('gisida')
+
+sampleLayer['id'] = 'sample-layer';
+sampleLayer['visible'] = false;
+layer1['visible'] = true;
+
+const initialState = {
+	"map-1": {
+		primaryLayer: "education",
+		showFilterPanel: false,
+		layers: {"sample-layer": {...sampleLayer}, "education": {...layer1}},
+		timeseries: {}
+	},
+}
```

Comment:
@ciremusyoka Let's make sure the tabbing in this file is all set to 2 spaces, pls 🙏